config/routes.rb

```diff
@@ -29,7 +29,7 @@
     get '/*route/:binary_name', to: 'folders/binaries#show', as: :binary, format: true
 
     post '/*route/:binary_name/like', to: 'folders/binaries/file_likes#create', as: :like_file, format: true
-    post '/*route/:binary_name/like_comment', to: 'folders/binaries/comment_likes#create', as: :like_comment, format: true
+    post '/*route/:binary_name/like_comment', to: 'folders/binaries/comment_likes#create', as: :like_comment
```

Comment:
to make this more RESTful - i might suggest `/*route/:binary_name/comments/:id/likes` - that way you don't have to pass the comment in manually.

same for the files route above `/*route/:binary_name/files/:id/likes`